packages/lib/src/nav-tabs/Tab.tsx

```diff
@@ -60,7 +59,7 @@ const DxcTab = forwardRef(
             </TabIconContainer>
           )}
           <DxcFlex alignItems="center" gap="0.5rem">
-            <BaseTypography
+            <LabelContainer
```

Comment:
I don't understand this implementation. Why aren't you consuming the theme from the provider as in the other styled-components?

I know it is an internal component without a `ThemeProvider` but it should only be used outside the NavTabs, which do have such a provider, so this component would have access to it.